cnv_release_notes/cnv_release_notes.adoc

```diff
@@ -11,65 +11,91 @@ include::modules/cnv_introduction_to_cnv.adoc[leveloffset=+2]
 [[RN_new_features]]
 == New and changed features
 
-==== Virtual machine control from the web console
-** The updated {ProductName} web console now allows users to create virtual 
-machines, access virtual machines with either a serial or VNC connection, and 
-perform basic power management tasks (start|stop) on virtual machines. 
+=== Introducing Operators
+* Operators power the installation of KubeVirt, Containerized Data Importer, and
+the *web-ui* in {ProductName} 1.4.
 
-==== Layer-2 networking
-** Layer-2 networking capabilities are now exposed to the {ProductName} 
-web console. Users can control the type of vNIC that is exposed to a virtual 
-machine, connect a virtual machine to an enterprise network defined by a VLAN 
-ID, and more.
+=== Updated KubeVirt API
+* A new version of the KubeVirt API is included in {ProductName} 1.4. Several
+important changes are reflected in the latest configuration file templates.
+** The `apiVersion` has been updated from `kubevirt.io/v1alpha2` to
+`kubevirt.io/v1alpha3`.
+** The `volumeName` attribute no longer exists. Ensure that each disk name
+matches the corresponding volume name in all configuration files.
+** All instances of `registryDisk` must be updated to `containerDisk` in
+configuration files.
 
-==== Upload existing virtual machine images into templates
-** You can upload an existing image into a template, which can then be used to 
-create new virtual machines.
 
-==== Clone virtual machines
-** You can clone the PVC of an existing virtual machine disk into a new 
-DataVolume, which can then be used to create a new virtual machine. 
+[[RN_resolved_issues]]
+== Resolved issues
+
+* The `runc` package version `runc-1.0.0-54` contained a bug that caused the
+virt-launcher to crash if FIPS was disabled. The version of `runc` containing
+the fix is now shipped with Red Hat Enterprise Linux 7 Extras.
+https://bugzilla.redhat.com/show_bug.cgi?id=1650512[(BZ1650512)]
+
+// Below issues are expected to be resolved, but may need double-checking
+
+* In the Create VM Wizard, using the *PXE* source option with the *Start VM on creation*
```

Comment:
s/Create VM/*Create Virtual Machine*/
Also on this line: the checkbox is *Start virtual machine on creation*